modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,615 @@
+  --from-file=MONGO_SERVER_URL=/home/user/MONGO_SERVER_URL 
+----
++
+.Example of configuring the `MONGO_SERVER_URL` value using a secret
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  chain:
+    disabled: false
+    storage.docdb.mongo-server-url-dir: /tmp/mongo-url
+    options:
+      deployments:
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: tekton-chains-controller
+                  volumeMounts:
+                  - mountPath: /tmp/mongo-url
+                    name: mongo-url
+                volumes:
+                -  name: mongo-url
+                   secret:
+                    secretName: mongo-url
+# ...
+----
+
+* With this update, when using KMS signing in {tekton-chains}, instead of providing the KMS authentication token value directly in the configuration, you can provide the token value as a secret by using the `signers.kms.auth.token-path` setting.
++
+To create a KMS token secret, enter the following command:
++
+[source,terminal]
+----
+$ oc create secret generic <secret_name> -n tekton-chains \
+  --from-file=KMS_AUTH_TOKEN=/home/user/KMS_AUTH_TOKEN <1>
+----
+<1> Replace `<secret_name>` with any name. The following example uses a KMS secret called `kms-secrets`.
++
+.Example configuration of the KMS token value using a secret called `kms-secrets`
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  chain:
+    disabled: false
+    signers.kms.auth.token-path: /etc/kms-secrets/KMS_AUTH_TOKEN
+    options:
+      deployments:
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: tekton-chains-controller
+                  volumeMounts:
+                  - mountPath: /etc/kms-secrets
+                    name: kms-secrets
+                volumes:
+                -  name: kms-secrets
+                   secret:
+                    secretName: kms-secrets
+# ...
+----
+
+* With this update, you can configure a list of namespaces as an argument to the {tekton-chains} controller. If you provide this list, {tekton-chains} watches pipeline runs and task runs only in the specified namespaces. If you do not provide this list, {tekton-chains} watches pipeline runs and task runs in all namespaces.
++
+.Example configuration for watching only the `dev` and `test` namespaces
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  chain:
+    disabled: false
+    options:
+      deployments:
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - args:
+                  - --namespace=dev, test 
```

Comment:
I have no idea, but I wondered if this this space after the comma between the namespaces would mess anything up, or if it's intentional syntax in this CR context. I saw what seemed like a similar instance in upstream docs where there was no comma, FWIW: https://tekton.dev/docs/chains/config/#example 

```
--namespace=dev,test
```